src/libs/DistanceRequestUtils.ts

```diff
@@ -78,6 +94,36 @@ function getRoundedDistanceInUnits(distanceInMeters: number, unit: Unit): string
     return convertedDistance.toFixed(2);
 }
 
+// TODO: I wonder if it would be better to refactor these functions to pass params in an object
```

Comment:
I would suggest to do it.

Reply:
@neil-marcellini wdyt? do you think it should be done in this PR?